### DISCUSSION TITLE
Recently added contributor user gets 0 results from the PNP search results

### DISCUSSION THREAD

**Discussion by allafridman** [microsoft-search/pnp-modern-search#3327] [General]
**Version used**
4.9.3.0

**Describe the bug**
Recently added contributor user gets 0 results from the PNP search results. Same user can use OOB site search. After a few hours the results are showing up using Pnp search results webpart.

**To Reproduce**
Add a  user to the site contributors group( the user did not have access before)
User searches using PNP search results webpart, 0 results ( search configured to search for that site)
User searches using out of the box search box for the site, gets many results
A few hours later PNP webpart returns results

**Expected behavior**
As soon as user is added to the contributors group of the site, the user should be able to search using Pnp webpart

**Comment by kasperbolarsen** [comment]
Hi @allafridman 
When you add an additional user, you will have to wait for the search  index to be updated, making the content available for the new user. The same happens when you add new content to a site collection. 
How often the search crawled will visit your site, will depend on a number of factors like how busy the farm the tenant resides in is.
It your tenant takes forever to get the search index updated you can try creating a support request with MS, but be aware that there isn't an official SLA on search index refresh rates.
I know enough about it to have my own early warning system,  https://www.youtube.com/watch?v=WWh6qF4UlJU
